src/Umbraco.Core/ContentApps/UserGroupEditorContentAppFactory.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using Umbraco.Cms.Core.Models.ContentEditing;
+using Umbraco.Cms.Core.Models.Membership;
+
+namespace Umbraco.Cms.Core.ContentApps
+{
+    public class UserGroupEditorContentAppFactory : IContentAppFactory
+    {
+        internal const int Weight = -100;
+
+        private ContentApp? _app;
+
+        public ContentApp? GetContentAppFor(object source, IEnumerable<IReadOnlyUserGroup> userGroups)
+        {
+            switch (source)
+            {
+                case IUserGroup _:
+                    return _app ??= new ContentApp()
+                    {
+                        Alias = "umbUserGroup",
+                        Name = "Detail",
```

Comment:
Same with this one - would be ideal to be localized

Reply:
Updated. The Aliasses are automatically used in translations, so I added the information to the translation file